pom.xml

```diff
@@ -139,19 +140,49 @@
             <version>4.13.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
add to changelog

Reply:
Was accidentally under "Plugins". Fixed.